src/sentry/models/featureadoption.py

```diff
@@ -50,7 +50,7 @@
 )
 manager.add(43, "user_tracking", "User Tracking", "code", prerequisite=["first_event"])
 manager.add(44, "custom_tags", "Custom Tags", "code", prerequisite=["first_event"])
-manager.add(45, "source_maps", "Source Maps", "code", prerequisite=["first_event", "javascript"])
+manager.add(45, "source_maps", "Source Maps", "code", prerequisite=["first_event", ("javascript", "node")])
```

Comment:
Does this work?

Reply:
Yes, see https://github.com/getsentry/sentry/blob/f4a19b6a423ac5a539a115e96b50591a931cc77b/src/sentry/models/featureadoption.py#L62

Reply:
Just some feedback here, but this is ultimately @ehfeng's decision:

I'm not sure I'd group node under this feature tracking flag thing. Source maps in node are generally less common and I wouldn't say "required" in the same sense as they are for browser js. And not sure if we'd want to say, using node sourcemaps also toggles this for browser js since I assume setup is going to be entirely different.